tools/analysis_tools/benchmark.py

```diff
@@ -105,14 +105,13 @@ def dataset_benchmark(args, cfg, distributed, logger):
 
 
 def main():
-    register_all_modules()
-
     args = parse_args()
     cfg = Config.fromfile(args.config)
-
     if args.cfg_options is not None:
         cfg.merge_from_dict(args.cfg_options)
 
+    init_default_scope(cfg.get())
```

Comment:
It should be uniformly written as init_default_scope(cfg.get('default_scope', 'mmdet'))